### PR TITLE
Changed some hardcoded value to properties

### PR DESCRIPTION
**Pull request by hanqidS2023**
Screenshot 2023-12-02 at 3 00 16 AM

### CODE REVIEW

File: src/components/RedistributeCard.jsx
Comment (xyang6230) [inline src/components/RedistributeCard.jsx:1]: 1. weight can be part of props
2. another onClickCard action would need to be passed in as a param
3. need to support image in this card (probably add as props.image)

**Comment by hanqidS2023** [comment]
Screenshot 2023-12-02 at 4 56 54 PM

File: src/components/RedistributeCard.jsx [hunk 1]
```diff
@@ -2,24 +2,38 @@ import React from "react";
...
+      <InfoCircleOutlined className="text-3xl text-white" />
     </div>
     <div className="flex flex-col w-64 mt-2">
-      <div className="mx-3">
-        <span className="text-primary font-bold text-lg font-heading">Pumpkin Pie</span>
+      <div className="mx-3" onClick={onClickRedistributeCard}>
+        <span className="text-primary font-bold text-lg font-heading">
+          {props.name}
+        </span>
         <p className="text-xs text-black text-primary font-paragraph">
```
Comment (congruiyS2023) [inline src/components/RedistributeCard.jsx]: `text-black` applies the same CSS property as `text-primary`, please only preserve one

File: src/components/RedistributeCard.jsx [hunk 2]
```diff
@@ -2,24 +2,38 @@ import React from "react";
...
+        </p>
+        <p className="text-xs text-black font-paragraph">
+          Status: {props.status}
+        </p>
+        <p className="text-xs text-black font-paragraph">
+          Pick up by {props.time}
         </p>
-        <p className="text-xs text-black font-paragraph">Untouched</p>
-        <p className="text-xs text-black font-paragraph">Pick up by 11/5/2023</p>
         <p className="text-xs text-black text-primary">
```
Comment (congruiyS2023) [inline src/components/RedistributeCard.jsx]: Same here, preserve one decoration to font color.

**Review by congruiyS2023 — CHANGES_REQUESTED** [review]
Looking good! Except for duplicate tailwind decorations on font color.

File: src/components/RedistributeCard.jsx (hunk 2 above)
Comment (congruiyS2023) [reply src/components/RedistributeCard.jsx]: Also on line 43 and line 46,  choose one between `text-primary` and `text-white`.

**Review by congruiyS2023 — APPROVED** [review]
Great!

**Pull request merged**